pos_pricelist/static/src/js/models.js

```diff
@@ -206,13 +206,8 @@ odoo.define("pos_pricelist.models", function (require) {
     models.Orderline.prototype.initialize = function (attr, options) {
         _Orderline_initialize.apply(this, arguments);
         if (options.product) {
-            this.set_unit_price(
-                options.product.price ||
```

Comment:
OK, I have digged through it and the fix is to remove `.product` in this line. Could you please push the new patch?

Reply:
you mean on line 210 replace
options.product.price ||
by
options.price ||
?
I just test the change and rewrite the commit.

Reply:
Yes, I mean that. :blush: